source/catalog.html

```diff
@@ -143,7 +153,7 @@ <h2 class="modal__title">Добавить в корзину</h2>
       <input class="form__input" type="radio" name="size" id="size-m" value="m">
       <label class="form__label" for="size-l">L</label>
       <input class="form__input" type="radio" name="size" id="size-l" value="l">
-      <button class="form__button button" type="submit">Добавить</button>
+      <button class="form__button form__button--modal button" type="submit">Добавить</button>
```

Comment:
```suggestion
      <button class="form__button button--color button" type="submit">Добавить</button>
```

Это мод не кнопки мобального окна или формы - это мод кнопки.